src/components/ConnectWalletButton.tsx

```diff
@@ -53,6 +54,11 @@ const ConnectWalletButton = () => {
     return 'Connect Wallet';
   })();
 
+  const handleSuggestChainAndConnectWallet = async () => {
+    await suggestChain(url);
```

Comment:
It might be better to have error handling, like showing a [toast](https://github.com/Agoric/dapp-inter/blob/main/src/service/wallet.ts#L141-L144), but given this only impacts testnets I don't think it's worth bothering.